pywwt_web/core.py

```diff
@@ -8,19 +8,23 @@
 
 
 class BaseWWTWidget(HasTraits):
-
+    
     def __init__(self):
-        super(BaseWWTWidget, self).__init__()
+        super(BaseWWTWidget, self).__init__() # does super() alone do the same?
```

Comment:
In Python 3 we could replace ``super(BaseWWTWidget, self)`` by ``super()`` but we need to be Python 2-compatible